cmd/bpf2go/main_test.go

```diff
@@ -406,6 +407,98 @@ func TestParseArgs(t *testing.T) {
 	})
 }
 
+func TestMultipleSourceFiles(t *testing.T) {
+	modRoot, err := filepath.Abs("../..")
+	if err != nil {
+		t.Fatal("Can't get module root:", err)
+	}
+
+	if _, err := os.Stat(filepath.Join(modRoot, "go.mod")); os.IsNotExist(err) {
```

Comment:
Can you move creating the temporary module into a helper instead of copying the code?